stdlib/2and3/ssl.pyi

```diff
@@ -219,6 +219,7 @@ class SSLSocket(socket.socket):
     def get_channel_binding(self, cb_type: str = ...) -> Optional[bytes]: ...
     def selected_alpn_protocol(self) -> Optional[str]: ...
     def selected_npn_protocol(self) -> Optional[str]: ...
+    def accept(self) -> Tuple[SSLSocket, Any]: ...
```

Comment:
Would use `socket._RetAddress` instead of `Any`. `_RetAddress` is currently `Any` but that might change perhaps. Also more informative.

Reply:
Updated PR as suggested.